models/connections.go

```diff
@@ -1,14 +1,63 @@
 package models
 
 import (
+	"bytes"
+	"context"
+	"fmt"
 	"net/url"
+	"os"
+	"os/exec"
 	"regexp"
 	"time"
 
 	"github.com/flanksource/duty/types"
 	"github.com/google/uuid"
 )
 
+// List of all connection types
+const (
+	ConnectionTypeAWS            = "AWS"
```

Comment:
I think lower_case these, and then toLower().replaceAll(" ", "_") the input for comparison